parallel-install/pkg/overrides/overrides.go

```diff
@@ -22,11 +29,24 @@ type Provider struct {
 	log                          func(format string, v ...interface{})
 }
 
+//OverridesProvider defines contract for reading overrides for given Helm release.
 type OverridesProvider interface {
+	//OverridesGetterFunctionFor returns a function returning overrides for a Helm release with provided name.
+	//Before using this function, ensure that the overrides cache is populated by calling ReadOverridesFromCluster function.
 	OverridesGetterFunctionFor(name string) func() map[string]interface{}
+
+	//Populates overrides cache by reading from the cluster. You have to call this function before using OverridesGetterFunctionFor.
 	ReadOverridesFromCluster() error
 }
 
+//New returns a new Provider.
+//
+//overridesYaml contains a list of manually-provided overrides.
+//every value in the list contains data in yaml format.
+//The structure of the file should follow Helm values.yaml convention.
+//There is one difference from plain Helm values.yaml: These are not values for a single release, but for entire Kyma installation.
+//Because of that you have to put values for a specific Component (e.g: Component name is "foo") under a key equal to component's name (i.e: "foo").
```

Comment:
```suggestion
//Because of that, you have to put values for a specific Component (e.g: Component name is "foo") under a key equal to the component's name (i.e: "foo").
```